images/run_workflows.sh

```diff
@@ -9,10 +9,28 @@ set -ex
 /usr/local/bin/checkout.sh /src
 
 # Trigger a workflow
-python -m kubeflow.testing.run_e2e_workflow \
-  --project=kubeflow-ci \
-  --zone=us-east1-d \
-  --cluster=kubeflow-testing \
-  --bucket=kubernetes-jenkins \
-  --config_file=/src/${REPO_OWNER}/${REPO_NAME}/prow_config.yaml \
-  --repos_dir=/src
+if [ -z "$CLOUD_PROVIDER" ]
+then
+  python -m kubeflow.testing.run_e2e_workflow \
+    --project=kubeflow-ci \
+    --zone=us-east1-d \
+    --cluster=kubeflow-testing \
+    --bucket=kubernetes-jenkins \
+    --config_file=/src/${REPO_OWNER}/${REPO_NAME}/prow_config.yaml \
+    --repos_dir=/src
+else
+  if [[ "$CLOUD_PROVIDER" == "aws" ]]
+  then
+    echo "Triggering AWS Argo Workflows"
+    python -m kubeflow.testing.run_e2e_workflow \
+      --cluster=${AWS_EKS_CLUSTER} \
+      --bucket=${ARTIFACTS_S3_BUCKET} \
+      --config_file=/src/${REPO_OWNER}/${REPO_NAME}/prow_config.yaml \
+      --repos_dir=/src \
+      --cloud_provider=aws \
+      --aws_region=${AWS_DEFAULT_REGION} \
+      --desired_node=${DESIRED_NODE} \
+      --min_node=${MIN_NODE} \
+      --max_node=${MAX_NODE}
+  fi
```

Comment:
why if `CLOUD_PROVIDER=gcp`?  Can you cover these edge cases?

Reply:
The point I didn't add `CLOUD_PROVIDER=gcp` is it require GCP folks to add `ENV CLOUD_PROVIDER=gcp` in their Dockerfiles.

Since it's AWS related PR, it might does not make sense to ask GCP to make changes. 

I can add them if @jlewi think it's okay to add so.

Reply:
Can we trigger the GCP logic if CLOUD_PROVIDER is not set or if it is gcp?

Reply:
Add below logic to handle GCP request:

```
if [ -z "$CLOUD_PROVIDER" ] ||  [ "$CLOUD_PROVIDER" = "gcp" ]
then
  python -m kubeflow.testing.run_e2e_workflow \
    --project=kubeflow-ci \
    --zone=us-east1-d \
    --cluster=kubeflow-testing \
    --bucket=kubernetes-jenkins \
    --config_file=/src/${REPO_OWNER}/${REPO_NAME}/prow_config.yaml \
    --repos_dir=/src
else

```

```
if not args.cloud_provider or args.cloud_provider == "gcp":
....
elif args.cloud_provider == "aws":
.... 
```